package.json

```diff
@@ -25,6 +25,7 @@
   "homepage": "https://github.com/VCityTeam/UD-Viz/",
   "peerDependencies": {},
   "dependencies": {
+    "d3": "^7.0.1",
```

Comment:
In order to keep track of package dependencies (e.g. only SPARQL uses d3, which can always be re-inferred by seaching "import d3" in all sources) with a comment in this file, maybe we could use this [json trick for npm](https://stackoverflow.com/a/45815391/5649243) ?